src/components/products/ProductSelector.tsx

```diff
@@ -22,6 +22,7 @@ interface Props {
   onClearButtonClick?: () => unknown;
   onProductCreated?: (product: Product) => unknown;
   onInputFocus?: () => unknown;
+  errorMessage?: string | string[];
```

Comment:
Is this correct? I thought it's always string[]?

Reply:
Hmm, I agree with you, there should always be an array of strings. But I just checked, wherever we have `errorMessage` as a prop, we have marked it as a `string | string[]`. I think now, should we do it as the rest of the app or can we do it here because we know it's the right way? The actual reason, I marked it like this, are other `errorMessage` props (on other custom components).

Reply:
Ah okies then, I though this was the case for this PR only. 

All good.